pySNOM/images.py

```diff
@@ -0,0 +1,219 @@
+import numpy as np
+import copy
+from enum import Enum
+import pySNOM.defaults as defaults
+
+MeasurementModes = Enum('MeasurementModes', ['None','AFM', 'PsHet', 'WLI', 'PTE', 'TappingAFMIR', 'ContactAFM'])
+DataTypes = Enum('DataTypes', ['Amplitude', 'Phase', 'Topography'])
+ChannelTypes = Enum('ChannelTypes', ['Optical','Mechanical'])
+
+# Full measurement data containing all the measurement channels
+class Data:
+    def __init__(self, filename = None, data = None, parameters = None, mode = "None"):
+        self.filename = filename  # Full path with name
+        # Measurement mode (PTE, PSHet, AFM, NanoFTIR) - Enum MeasurementModes
+        self.mode = MeasurementModes[mode]
+        # Full data from the gwy files with all the channels
+        self.data = data # All channels - Dictionary
+        # Other parameters from info txt -  Dictionary
+        if parameters is not None:
+            self.setParameters(parameters)
+        else:
+            self.parameters = parameters
+
+    # Method to set the actual data read from the gwyddion or gsf files
+    def setData(self, data: dict):
+        self.data = data
+
+    # Method to set measurement mode directly
+    def setMeasurementMode(self, measmode: str):
```

Comment:
For such things python people tend to use properties: https://docs.python.org/3/library/functions.html#property

But are these setters really needed? If this class is mostly (and likely should) be used as initialized, they do not make any sense are just a bunch of dead code.